packages/legacy/core/App/localization/fr/index.ts

```diff
@@ -455,6 +455,8 @@ const translation = {
         "ProofAccepted": "Preuve acceptée",
         "ProofRejected": "Preuve rejetée",
         "RequestedCredentialsCouldNotBeFound": "Les justificatifs demandés n'ont pas été trouvés",
+        "PresentationMetadataNotFound": "N'a pas pu trouver les métadonnées de présentation",
+        "CredentialMetadataNotFound": "Impossible de trouver les métadonnées des justificatifs",
```

Comment:
Did you put these in google translate / chatgpt? Maybe good if someone can review them in that case

Reply:
Yes would be good if some can review this